### PR TITLE
DT-6592 Navigator bottom sheet for envs without ticket sales

### PR DESCRIPTION
**Pull request by partisaani**
## Proposed Changes
Refactored the bottom card to match designs

## Pull Request Check List

  - A reasonable set of unit tests is included
  - Console does not show new warnings/errors
  - Changes are documented or they are self explanatory
  - This pull request does not have any merge conflicts
  - All existing tests pass in CI build

## Review

  - Read and verify the code changes
  - Test the functionality by running the UI locally with all popular browsers available in your platform
  - Check that the implementation matches the design, when such one is defined in a Jira issue
  - Merge the pull request

### CODE REVIEW

File: app/component/itinerary/navigator/NaviBottom.js [hunk 1]
```diff
@@ -8,35 +9,47 @@ export default function NaviBottom(
...
+          values={{ min: remainingDuration }}
+        />
+      </span>
+      <span className="navi-daytime">{epochToTime(arrival, config)}</span>
+    </div>
+  );
+
+  const [FirstElement, SecondElement] = isTicketSaleActive
+    ? [closeButton, durationDiv]
+    : [durationDiv, closeButton];
```
Comment (partisaani) [inline app/component/itinerary/navigator/NaviBottom.js:42]: Not sure how happy I am with this logic and variable naming. Definitely open for suggestions.

File: app/component/itinerary/navigator/NaviBottom.js [hunk 2]
```diff
@@ -8,35 +9,47 @@ export default function NaviBottom(
   { setNavigation, arrival, time },
   { config },
 ) {
+  const handleClose = () => setNavigation(false);
+  const handleTicketButtonClick = e => e.stopPropagation();
```
Comment (partisaani) [inline app/component/itinerary/navigator/NaviBottom.js]: Refactored function calls away from template to prevent them from being recreated on every render

Comment (vesameskanen) [reply app/component/itinerary/navigator/NaviBottom.js]: Hmm, It seems that these consts are defined as often as the component renders. stopPropagation could be moved outside the component function, though.

File: app/component/itinerary/navigator/NaviBottom.js (hunk 1 above)
Comment (vesameskanen) [reply app/component/itinerary/navigator/NaviBottom.js:42]: It is OK in my opinion.

File: app/component/itinerary/navigator/NaviBottom.js [hunk 3]
```diff
@@ -8,35 +9,47 @@ export default function NaviBottom(
...
+    </div>
+  );
+
+  const [FirstElement, SecondElement] = isTicketSaleActive
+    ? [closeButton, durationDiv]
+    : [durationDiv, closeButton];
+
   return (
     <div className="navi-bottom-sheet">
-      <div className="navi-bottom-controls">
```
Comment (vesameskanen) [inline app/component/itinerary/navigator/NaviBottom.js]: I guess these two divs can be combined. I wonder if navi-bottom-sheet class is currently unnecesary.

**Review by vesameskanen — CHANGES_REQUESTED** [review]

File: app/component/itinerary/navigator/NaviBottom.js (hunk 3 above)
Comment (partisaani) [reply app/component/itinerary/navigator/NaviBottom.js]: Combining.

File: app/component/itinerary/navigator/NaviBottom.js (hunk 2 above)
Comment (partisaani) [reply app/component/itinerary/navigator/NaviBottom.js]: Nothing a useCallback couldn't fix

**Pull request merged**